api/OffscreenCanvas.json

```diff
@@ -0,0 +1,639 @@
+{
+  "api": {
+    "OffscreenCanvas": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/OffscreenCanvas",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "60",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "<code>Experimental canvas features</code>"
```

Comment:
The `name` property of a flag object should not contain `\<code>` tags. Here and throughout the whole file.
```diff
- "name": "<code>Experimental canvas features</code>"
+ "name": "Experimental canvas features"
```

Reply:
oops. sorry :)